databricks/koalas/indexes.py

```diff
@@ -2077,7 +2088,7 @@ def holds_integer(self):
         """
         return isinstance(self.spark.data_type, IntegralType)
 
-    def item(self):
+    def item(self) -> Union[Scalar, Tuple]:
```

Comment:
Actually using `Scalar` is strictly not correct because this can return a pandas instance too. For exmaple, `pd.Timestamp` which is not in the `Scalar` from my cursory reading. But I think it's okay - we can fix it later separately.

Reply:
This makes sense. How about we define a pandas Scalar separately? @HyukjinKwon